ci/plugins/mzcompose/hooks/command

```diff
@@ -51,6 +51,14 @@ if ! mzcompose --mz-quiet list-workflows | grep -q "$service"; then
     mzcompose up -d --scale "$service=0" "$service"
 fi
 
+ci_uncollapsed_heading ":docker: Fetching binaries for coverage"
+mzcompose create
+mkdir -p coverage/
+chmod 777 coverage/
+mzcompose cp materialized:/usr/local/bin/environmentd coverage/ || true
+mzcompose cp materialized:/usr/local/bin/clusterd coverage/ || true
+mzcompose cp sqllogictest:/usr/local/bin/sqllogictest coverage/ || true
+
```

Comment:
You need `down -v` here, to destroy the containers you just started -- this is expecially important since the materialized containers will perform database bootstrap and whatnot.

Reply:
See above, I think not required.